### PR TITLE
Add missing eslint dependency to readme.md

### PR DESCRIPTION
**Pull request by gaborluk**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ An [ESLint shareable config](https://eslint.org/docs/developer-guide/shareable-c
 ## Usage
 
 ```
-npm install --save-dev eslint-plugin-standard eslint-plugin-promise eslint-plugin-import eslint-plugin-node @typescript-eslint/eslint-plugin eslint-config-standard-with-typescript
+npm install --save-dev eslint eslint-plugin-standard eslint-plugin-promise eslint-plugin-import eslint-plugin-node @typescript-eslint/eslint-plugin eslint-config-standard-with-typescript
```
Comment (mightyiam) [inline readme.md]: Thank you for the PR, @gaborluk.

In general, this seems desired, to me.

A few lines below, there's a list explaining this long list of dependencies. Could you please add at the beginning `1. ESLint.`?

**Comment by gaborluk** [comment]
I saw the errors, @mightyiam, but I don't know what `subject` and `type`, which are supposed to be non-empty, are. Could you please clarify?

Also, I will not be able to amend my previous commits as I've sent them through GitHub's web interface. Would you like me to close this PR, and send a new one?

**Comment by mightyiam** [comment]
@gaborluk regarding the commit message syntax:
https://www.conventionalcommits.org

Regarding amending the commits; it is possible. It's just a branch with commits in it:
https://github.com/gaborluk/eslint-config-standard-with-typescript/tree/patch-1

You can clone your fork, checkout the branch and amend the commits.

Or make a new one, if you prefer.

**Comment by gaborluk** [comment]
Thank you for the guidance, @mightyiam. I apologize for not recognizing this convention.

All should be fine now.

**Comment by mightyiam** [comment]
Thank you for your contribution, @gaborluk !

**Comment by gaborluk** [comment]
Sure thing, @mightyiam!

**Pull request merged**